src/components/Community/CommunitySection.tsx

```diff
@@ -78,7 +86,35 @@ export const CommunitySection: FC<CommunitySectionProps> = ({
           {isFeatureEnabled(FEATURE_COMMUNICATIONS) && (
             <>
               <TabPanel classes={{ root: styles.tabPanel }} value={'updates'}>
-                <Updates messages={updates} />
+                <Box maxHeight={600} style={{ overflowY: 'auto', overflowX: 'clip' }}>
+                  {updates && (
+                    <AvatarsProvider users={users}>
+                      {detailedUsers => (
+                        <CommunityUpdatesView
+                          entities={{
+                            members: detailedUsers,
+                            messages: updates,
+                          }}
+                          actions={{
+                            onSubmit: (_: string) => {
```

Comment:
What is the purpose of this handler?

Reply:
Was intended to serve a placeholder if we enable admins to write posts from the community section, but we can drop it for now.